webcompat/static/css/src/form.css

```diff
@@ -76,6 +76,7 @@
   display: block;
   width: 100%;
   padding: 4px 0;
```

Comment:
Would you please be able to add `padding-right` to the line above?

This would look like:
`padding: 4px 25px 4px 0` 

Thanks!